### ISSUE TITLE
[$20] Social media icons do not have appropriate labels.

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3099]
**Steps to Reproduce**
1. Launch the url https://www.topcoder.com/community/accessibility-testing. 2. Ensure that the screen reader is switched on. 3. Navigate to the social media icons by arrow keys. 4. These are read as 'icon facebook' 'icon twitter' etc.

**Expected Result**
The icons should be read as 'facebook' 'twitter' etc. There should be a visible text for these social media icons.

**Actual Result**
Social media icons do not have visible text associated with them.

**Device**: Desktop/Labtop
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=d201e9b0-60ca-4f87-97bc-5041dbbfc8f1&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft-copilots_topcodermsft_onmicrosoft_com%2FDocuments%2F[FAST] Test for Accessibility Series%2Fkameshwarid%2Fbug7

**Comment by kkartunov** [comment]
@topcoder-platform/topcodercompetitors this is now open for pickup

**Comment by Gauravseta** [comment]
@kkartunov PR on?

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097742 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097742 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by kkartunov** [comment]
`qa-accessibility` thanks

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097742 has been updated - it has been assigned to gauravseta.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by Gauravseta** [comment]
@kkartunov PR https://github.com/topcoder-platform/community-app/pull/3138

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097742<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by drasticdpk** [comment]
Verified on Windows/Chrome.
